routes/api/auth.js

```diff
@@ -0,0 +1,38 @@
+const router = require('express').Router();
+const config = require('./../../config');
+const jwt = require('jsonwebtoken');
+const { crudService, errorService } = require('./../../services');
+const { validationMiddleware } = require('./../../middlewares');
+const { UserModel } = require('./../../models');
+const { SCHEMAS, SCHEMAS_TYPE } = require('./../../shared/constants');
+
+router.post(
+    '/login',
+    validationMiddleware(SCHEMAS.USER, SCHEMAS_TYPE.READ),
+    (req, res) => {
+        const errorMsg = 'Invalid Password / User Name';
+
+        crudService
+            .findByKey(UserModel, res, { firstName: req.body.firstName} , false)
+            .then(user => {
+                user.comparePassword(req.body.password, (error, isMatch) => {
```

Comment:
what if user not found?

Reply:
Everything is OK. Working catch block and response return error message "Invalid Password / User Name"